### PR TITLE
PD-2758: Add Slide-tags docker

### PR DESCRIPTION
**Pull request by aawdeh**
Slide-tags Dockerfile taken from https://github.com/MacoskoLab/Macosko-Pipelines/blob/main/positioning/Dockerfile

 Modified slightly to reduce the number of layers, etc

### CODE REVIEW

File: 3rd-party-tools/slide-tags/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Taken from https://github.com/MacoskoLab/Macosko-Pipelines/blob/main/positioning/Dockerfile
+# Modified slightly to reduce the number of layers
+
+FROM debian:bookworm
+SHELL ["/bin/bash", "-c"]
+
+LABEL MAINTAINER="Broad Institute DSDE <dsde-engineering@broadinstitute.org"
```
Comment (ekiernan) [inline 3rd-party-tools/slide-tags/Dockerfile]:
```suggestion
LABEL maintainer="Broad Institute Pipeline Development Team <pipeline-development@broadinstitute.org"
```

**Review by ekiernan — APPROVED** [review]
Thank you! Just a suggestion on the label for the docker based on our recent warp-tools one

File: 3rd-party-tools/slide-tags/spatial-count.jl [hunk 2]
```diff
@@ -0,0 +1,561 @@
...
+    create_group(file, "metadata/SB_HD")
+    file["metadata/SB_HD/type"] = keys(metadata["SB_HD"]) |> collect
+    file["metadata/SB_HD/count"] = values(metadata["SB_HD"]) |> collect
+    
+    file["metadata/downsampling"] = downsampling # Vector{UInt32}
+end;
+
+println("done") ; flush(stdout) ; GC.gc()
+
+##### Documentation ############################################################
```
Comment (khajoue2) [inline 3rd-party-tools/slide-tags/spatial-count.jl]: Do we want to keep this in the same code doc or make a .md file?

File: 3rd-party-tools/slide-tags/spatial-count.jl [hunk 3]
```diff
@@ -0,0 +1,561 @@
...
+using ArgParse
+using CodecZlib
+using IterTools: product
+using StatsBase: countmap, sample
+using DataFrames
+using StringViews
+using LinearAlgebra: dot
+using Combinatorics: combinations
+
+# This file was taken from https://github.com/MacoskoLab/Macosko-Pipelines/blob/main/spatial-count/spatial-count.jl
```
Comment (khajoue2) [inline 3rd-party-tools/slide-tags/spatial-count.jl]: acknowledge the source of the code explicitly in a "Credits" or "Attributions" section ideally in a separate .md file

**Review by khajoue2 — APPROVED** [review]

File: 3rd-party-tools/slide-tags/spatial-count.jl (hunk 2 above)
Comment (aawdeh) [reply 3rd-party-tools/slide-tags/spatial-count.jl]: Were not using this code -- I had it for testing. Were pulling the script from their repo. Im going to remove it

File: 3rd-party-tools/slide-tags/spatial-count.jl (hunk 3 above)
Comment (aawdeh) [reply 3rd-party-tools/slide-tags/spatial-count.jl]: This script was added here for testing

Comment (aawdeh) [reply 3rd-party-tools/slide-tags/spatial-count.jl]: I removed the file

**Comment by aawdeh** [comment]
Created new ticket to deal with the Trivy errors in the PR

**Pull request merged**